.github/workflows/nightly.yml

```diff
@@ -0,0 +1,125 @@
+name: Nightly
+
+on:
+  push:
+  schedule:
+    - cron: 0 0 * * *
+
+
+jobs:
+  generate-proving-keys:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install --yes \
+            build-essential \
+            libgmp-dev \
+            libsodium-dev \
+            nasm
+
+      # # To prevent `npm install` failure of circuit package, 
+      # # it has to checkout manually because version of `circomlib` has pinned to a specific commit
+      # - name: Checkout circomlib
```

Comment:
Why is this now commented out?

If not needed, can we delete this?

Reply:
Removed in 20676ac36afb18c8d763b05e6d680aec52f0c518